documentation/ADRs/010_evaluation_metrics.md

```diff
@@ -0,0 +1,55 @@
+| Coverage (Histograms)               | -                     | Probabilistic    | Histogram-based measure of prediction coverage                                                             |
+| Sinkhorn/Earth-mover Distance & pEMDiv| Sinkhorn/EMD & pEMDiv | Probabilistic | Measures the difference between two probability distributions by calculating the minimal cost to transform one distribution into another |
+| Variogram                           | -                     | Probabilistic    | Measures spatial dependence in probabilistic models                                                        |
+| Average Precision                   | AP                    | Classification   | Measures the precision-recall trade-off in classification tasks                                             |
+| Root Mean Squared Logarithmic Error | RMSLE                 | Regression       | Evaluates the error between predicted and actual values, particularly suited for skewed data                |
+| Pearson                             | -                     | Regression       | Evaluates the linear correlation between two variables                                                      |
+
+
+## Consequences
+**Positive Effects:**
+
+- **More Accurate Model Evaluation**: By using metrics that account for the skewness and zero-inflation in the data, model performance can be more accurately assessed.
+- **Improved Onset Detection**: Focusing on metrics that emphasize onset detection ensures models are evaluated on their ability to predict critical shifts in conflict.
+
+**Negative Effects:**
+
+- **Difficulty in Interpretation**: Some metrics may require additional expertise to interpret and understand, 
+potentially making it harder to communicate results to non-technical.
+
+## Rationale
+The selected metrics are designed to address the unique characteristics of conflict prediction data, which tends to be zero-inflated and right-skewed. 
+Relying solely on traditional error metrics such as MSE (MSLE) can result in poor performance on relevant tasks like identifying onsets of conflict.
+
+Using a mix of probabilistic and point-based metrics will allow us to:
+- Better capture the range of possible outcomes and assess predictions in terms of uncertainty.
+- Focus evaluation on onsets of conflict, which are often the most critical and hardest to predict.
+- Ensure consistency and calibration across different spatial and temporal resolutions, from grid-level to country-level predictions.
+
+### Considerations
+- **Skewed Data**: The distribution of fatalities is often zero-inflated and right-skewed, which makes evaluation more challenging. Metrics must account for this distribution to avoid favoring models that predict zeros too often.
+- **Onset Sensitivity**: The ability to detect onsets of conflict is particularly important, as these events are often of the most interest to decision-makers.
```

Comment:
We'll need to be more specific here, but don't worry about that for now - we'll do that before we send it to review by itself